main.go

```diff
@@ -1,39 +1,52 @@
 package main // import "moul.io/moulsay"
 
 import (
+	"context"
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 	"strings"
 
-	cli "gopkg.in/urfave/cli.v2"
+	"github.com/peterbourgon/ff/v3/ffcli"
 	"moul.io/moulsay/moulsay"
 )
 
 func main() {
-	app := cli.App{
-		Flags: []cli.Flag{
-			&cli.IntFlag{Name: "max-width", Aliases: []string{"w"}, Value: 72},
+	fs := flag.NewFlagSet("app", flag.ExitOnError)
+	maxWidth := fs.Int("max-width", 72, "set max width")
```

Comment:
```suggestion
	var (
		fs =       flag.NewFlagSet("app", flag.ExitOnError)
		maxWidth = fs.Int("max-width", 72, "set max width")
	)
```

about flags, I prefer this pattern